databuilder/databuilder/extractor/delta_lake_metadata_extractor.py

```diff
@@ -299,7 +299,7 @@ def fetch_columns(self, schema: str, table: str) -> List[ScrapedColumnMetadata]:
                 field_dict[field.name] = field
         except AnalysisException as e:
             LOGGER.error(e)
-            return raw_columns
+            return None
```

Comment:
another option would be to return `[]` here; i thought `None` felt a bit more explicit about failure versus an empty list, but i'm open to whatever would be more consistent with the codebase broadly